CONTRIBUTING.md

```diff
@@ -158,3 +223,319 @@ including how to setup a local Kubernetes cluster, can be found in
 5. Create a Pull Request on GitHub.
    Visit your fork at `https://github.com/vmware-tanzu/antrea` and click
    `Compare & Pull Request` button next to your `remoteRevertName` branch.
+
+## Issue Management
+
+We use labels and workflows (some manual, some automated with GitHub Actions) to
+help us manage triage, prioritize, and track issue progress. 
+
+### Issue Workflow
+
+The purpose of this workflow is to formalize a lightweight set of processes that
+will optimize issue triage and management which will lead to better release
+predictability and community responsiveness for support and feature
+enhancements. Additionally, Antrea must prioritize issues to ensure interlock
+alignment and compatibility with other projects including Kubernetes. The
+processes described here will aid in accomplishing these goals.
+
+![developer workflow overview](docs/assets/developer-workflow-opaque-bg.png)
+
+For additional details and to improve the metadata associated with your issues,
+please keep reading.
+
+### New Issues
+
+When submitting a new issue, please search the existing set of issues
+to see if an issue has already been created that addresses your concern.
+
+When creating your new issue, please ensure that you:
+
+* label the issue with the appropriate [`kind/<?>`](#kinds) label.  The kind of
+  issue will determine the issue workflow.
+* label the issue with the appropriate [`area/<?>`](#area-assigment) label. The
+  area label will identify the area of interest an issue or PR belongs in and
+  will ensure the appropriate reviewers shepherd the issue or PR through to its
+  closure.
+* for issues, label the estimated size (if known). The size can be updated
+  during backlog grooming by the contributors. This estimate is used to guide
+  sprint scoping the exercise.
```

Comment:
maybe I'm ignorant, but what does "sprint scoping the exercise" mean?

Reply:
typo... will fix.